group_vars/all.yml

```diff
@@ -135,4 +135,6 @@ disable_firewall: false
 ## host provision variables
 instance_ssh_user: "{{ lookup('env','INSTANCE_SSH_USER')|lower|default('ansible',true) }}"
 firsttime_connect_user: "customeradmin"
-control_node_key_file: "~/.ssh/id_rsa_bms_toolkit"
\ No newline at end of file
+control_node_key_file: "~/.ssh/id_rsa_bms_toolkit"
+proxy_setup: "{{ lookup('env','ORA_PROXY_SETUP')|lower|default('false',true) }}"
```

Comment:
Would it be possible to have some sort of validation, post-setup (especially if proxy_setup was false), to give a clear error message when required Internet connectivity (OS packages?) doesn't exist?

Reply:
Added a new task as validation for RHSM layer where we need to ensure internet connectivity to the subscription server.
The rpms per se are installed as needed by the `install-oracle.sh`